### ISSUE TITLE
Can the CDN support non-NPM registries?

### ISSUE THREAD

**Issue by zachsa** [jspm/project#290]
I understand that the CDN basically tracks NPM to transpile all published packages to be in a suitable module format made available via the CDN - let me know if this is not correct.

I would like to publish an NPM package to the GitHub registry, that I can then consume via the JSPM CDN. Is this possible? It would be cool if there was a UI where I could 'register' a package that I would like to consume via the JSPM CDN.

**Comment by trusktr** [comment]
Yeah, being able to make an account, then register some static file location for some package (f.e. a git repo), could be an interesting idea. It would allow people to publish packages in form of ES modules from anywhere. It would be similar to how Deno currently supports GitHub repos (I don't know if it supports others).
